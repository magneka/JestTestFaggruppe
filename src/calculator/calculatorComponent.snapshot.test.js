import React from 'react';
import { render } from '@testing-library/react';
import CalculatorComponent from './calculatorComponent';

it('Calculatorcomponent snapshot test', () => {
    const container = render(<CalculatorComponent />)
    expect(container.baseElement).toMatchSnapshot()    
});