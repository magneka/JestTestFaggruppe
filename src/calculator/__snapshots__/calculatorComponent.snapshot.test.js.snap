// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Calculatorcomponent snapshot test 1`] = `
<body>
  <div>
    <form>
      <div>
        <input
          data-testid="a"
          name="a"
          size="4"
          type="text"
          value=""
        />
        <select
          data-testid="operator"
        >
          <option
            value=""
          />
          <option
            value="+"
          >
            +
          </option>
          <option
            value="-"
          >
            -
          </option>
          <option
            value="/"
          >
            /
          </option>
          <option
            value="*"
          >
            *
          </option>
        </select>
        <input
          data-testid="b"
          name="b"
          size="4"
          type="text"
          value=""
        />
        <input
          data-testid="submit"
          type="submit"
          value="="
        />
        <input
          data-testid="result"
          name="res"
          readonly=""
          size="6"
          type="text"
          value=""
        />
      </div>
    </form>
  </div>
</body>
`;
